libraries/aws_s3_bucket.rb

```diff
@@ -38,7 +38,7 @@ def public?
   def has_access_logging_enabled?
     return unless @exists
     # This is simple enough to inline it.
-    !AwsS3Bucket::BackendFactory.create.get_bucket_logging(bucket: bucket_name).logging_enabled.nil?
+    !AwsS3Bucket::BackendFactory.create(inspec_runner).get_bucket_logging(bucket: bucket_name).logging_enabled.nil?
```

Comment:
should this just be `!BackendFactory.create(inspec_runner)`? Both work but I saw you removed the others.

Reply:
It's just more concise without the prefix.  I think I started removing the prefixes and got tired.  I can make them consistent.